src/bin/exrcheck/main.cpp

```diff
@@ -30,6 +30,7 @@ usageMessage (const char argv0[])
     cerr << "  -m : avoid excessive memory allocation (some files will not be fully checked)\n";
     cerr << "  -t : avoid spending excessive time (some files will not be fully checked)\n";
     cerr << "  -s : use stream API instead of file API\n";
+    cerr << "  -v : print version info\n";
```

Comment:
Would it be better if this said 'OpenEXR and IMath version info' so it's clear it isn't outputting the version of the provided file, as exrheader does?

Reply:
Good point, done.